packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -173,8 +176,9 @@ export default class SelectOpener extends React.Component<
                 aria-haspopup="listbox"
                 data-testid={testId}
                 id={id}
+                role="combobox"
```

Comment:
Side note: Since our selects now use `role=combobox`, we'll be able to treat SingleSelect/MultiSelect more like fields! I created WB-1807 for applying aria-required on these components since it's only valid on `role=combobox` and not `role=button` 😄

Reply:
LOVE that! And yes, `combobox` makes it into a real form field with a value! (even though VoiceOver is buggy on that part!)